### PR TITLE
Fix PointCNN benchmark; sklearn is replaced by scikit-learn.

### PR DESCRIPTION
**Pull request by aywi**
Fix PointCNN benchmark (dim (int) is required); [sklearn](https://pypi.org/project/sklearn/) is replaced by [scikit-learn](https://pypi.org/project/scikit-learn/).

### CODE REVIEW

**Comment by rusty1s** [comment]
Thank you!
